test/webhook_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+kubectl apply  &> /dev/null -f correctconfig.yaml
+if [ $? -eq 0 ]; then
+    VAR1="pass"
+    echo pass 
+else
+    echo "Test Failed: " 
+    kubectl apply -f correctconfig.yaml
```

Comment:
I think you want to get rid of this line, and add some info more to the `echo` above.